src/packages/DAO/elections.dao.ts

```diff
@@ -51,18 +60,38 @@ export async function startElection(formData: FormData) {
   redirect(`elections/${electionId}`);
 }
 
+async function getElections() {
+  const res = await prisma.election.findMany();
+  if (!res) {
+    throw new NotFound("");
+  }
+  return res;
+}
+async function findByNameOrStatus({ name, status }: FindElection) {
+  const resultElection = await getElections();
+  const result = resultElection.find(
+    (e) => e.name === name && e.status === status,
```

Comment:
comment: this is why we use `id`, to avoid comparing two or more things :)

nothing to change here, unless you wanna refactor your `getElections`

Reply:
In case you have a suggestion on how to refactor, I can do it without any problem. I might be wrong, but based on the ID, wouldn’t I need to compare the incoming values with the values of an existing ID? Maybe the method I thought of, using the ID, is being influenced by the response I suggested here

Reply:
no worries, leave this to another task, or else this can get our of hand